packages/rum/test/userActionCollection.spec.ts

```diff
@@ -92,6 +103,54 @@ describe('startUserActionCollection', () => {
     button.click()
 
     clock.expire()
+  }
+
+  it('cancels user action on view loading', () => {
```

Comment:
we should have two cases:
- cancel pending user action on new view
- dismiss new user action during view loading